backend-project/small_eod/letters/tests.py

```diff
@@ -13,3 +17,55 @@ class LetterFactoryTestCase(FactoryCreateObjectsMixin, TestCase):
 class DescriptionFactoryTestCase(FactoryCreateObjectsMixin, TestCase):
     FACTORY = DescriptionFactory
     MODEL = Description
+
+
+class PresignedUploadFileTestCase(APITestCase):
+    def test_getting_form_data(self):
+        url = reverse('file_upload')
+        data = {
+            'name': 'text.file',
+        }
+
+        response = self.client.post(url, data, format='json')
+        print(response.data)
```

Comment:
Czy pozostawiłeś to celowo?

Reply:
Tak. W zasadzie to draft, jeśli do POST nie ma nic przeciwko, to dokończę test sprawdzając klucze w "formData" i z poprawię dokumentacje.

Reply:
Dokumentacja mile widziana.